docs/data_collection.rst

```diff
@@ -77,41 +77,52 @@ with information about the user's browser.
 Metrics
 -------
 
-Snippet code, which is executed on about:home, sends HTTP requests to a server
-located at https://snippets-stats.moz.works and/or
-https://snippets-stats.mozilla.org whenever an event occurs that we would like
-to measure. These requests are sampled at a rate of 1%, meaning that only 1% of
-the time an event occurs will a request be made.
-
-Requests sent to snippets-stats.mozilla.org contain the following data (sent as
-URL parameters in the query string) in addition to the normal data available
-from an HTTP request:
-
-Snippet Name
-   Unique name referring to the snippet that was being viewed when the request
-   was sent.
+By default, the about:newtab and about:home pages in Firefox (the pages you see when you
+open a new tab and when you start the browser), will send data back to Mozilla servers
+about snippets. The intent is to collect data in order to improve the user's experience
+while using Firefox. At any time, it is easy to **turn off** this data collection by
+`opting out of Firefox telemetry <https://support.mozilla.org/kb/share-telemetry-data-mozilla-help-improve-firefox>`_.
+
+Data is sent to our servers in the form of discreet HTTPS 'pings' or messages whenever
+you see or interact with snippets on about:home or about:newtab. We try to minimize the
+amount and frequency of pings by batching them together.
+
+At Mozilla, `we take your privacy very seriously <https://www.mozilla.org/privacy/>`_.
+The new tab page will never send any data that could personally identify you. We do not
+transmit what you are browsing, searches you perform or any private settings.
+Activity Stream does not set or send cookies, and uses
+`Transport Layer Security <https://en.wikipedia.org/wiki/Transport_Layer_Security>`_ to
+securely transmit data to Mozilla servers.
+
+Data collected about snippets is retained on Mozilla secured servers for a period of
+30 days before being rolled up into an anonymous aggregated format.  After this period
+the raw data is deleted permanently. Mozilla **never shares data with any third party**.
+
+The following is an overview of the different kinds of data we collect about snippets:
+
+Message ID
```

Comment:
Do we need to include client_id etc. here? What do you think?

Reply:
I don't think so. Instead, we can add a link to [Firefox Privacy Notice](https://www.mozilla.org/en-US/privacy/firefox/), which already covers data collection for Snippets in the "Suggested relevant content" section.